planner/core/src/main/java/eu/seaclouds/platform/planner/core/Planner.java

```diff
@@ -296,32 +309,31 @@ public SLAInfo() {}
 
     }
 
-    class OfferJson {
+    public static class Offering {
```

Comment:
A public inner class will be accessible be the outer class so no reason to be defined as nested class.
In fact, `Offering` can be reused by Matchmaker, Discoverer and possibly Optimizer, just to mention the first components that come to my mind: I hope you see why it is conceptually not an inner class.